src/sage/coding/channel.py

```diff
@@ -31,15 +32,24 @@
 
 This file contains the following elements:
 
-    - :class:`Channel`, the abstract class for Channels
+    - :class:`Channel`, the abstract class for channels
     - :class:`StaticErrorRateChannel`, which creates a specific number of errors in each
       transmitted message
     - :class:`ErrorErasureChannel`, which creates a specific number of errors and a
       specific number of erasures in each transmitted message
+    - :class:`StaticRankErrorChannel`, which creates an error of specific rank in each
+      transmitted message. This is the rank analogue of :class:`StaticErrorRateChannel`
+
+AUTHORS:
+
+ - David Lucas (2015): initial version
+ - Maxime Bombar (2021): Rank metric channel
+ - Julien Lavauzelle (2025): Fix documentation and minors issues on rank metric channels
 """
```

Comment:
No leading spaces in the lists above. If you check the rendered doc, they look ugly.